src/docs/product/sentry-basics/search/searchable-properties/session-replay.mdx

```diff
@@ -120,6 +120,12 @@ The number of URLs that the user visited during a replay recording.
 
 - **Type:** number
 
+### `dead.selector`
+
+Similar to the `click.selector` search property, but only queries on dead clicks. An element identified using a subset of CSS selector syntax. For example, `span#section-1` or `span.active` or `span[role=button]` or `span#section-1.active[role=button]` would all match the element `<span id="section-1" class="active" role="button"/>`. Note that, CSS combinators, pseudo selectors, and attr selectors other than `=` are not supported.
```

Comment:
is the node type, `span`, required?

Reply:
<img width="609" alt="SCR-20230928-loxa" src="https://github.com/getsentry/sentry-docs/assets/56095982/022147f4-fb93-4ae3-bb2c-9fc7e41d8c3f">
doesn't seem like it

Reply:
Maybe give a mix of suggestions to give user an idea that you *can* use the el type

> `#section-1` or `span.active` or `span[role=button]` or `.active[role=button]`

Reply:
Ya, more mixed selectors is what i was thinking. I should've just said that, don't mean to be passive aggressive sounding